setup.py

```diff
@@ -46,8 +46,10 @@
       scripts=glob('scripts/*'),
       extras_require={'test': ["nose >= 0.10.1", "pep8"],
                       'doc': ["Sphinx >= 1.2.2", "sphinx-bootstrap-theme"]},
-      install_requires=['click', 'numpy >= 1.7',
+      install_requires=['click == 7.0-dev', 'numpy >= 1.7',
```

Comment:
Click 7.0 will include option to hide commands and options from help (see [PR](https://github.com/pallets/click/pull/500)) which I wanted for parallel_deblur. We can switch to 7.0 when it's released?